modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -0,0 +1,118 @@
+// Module included in the following assemblies:
+//
+// * logging/efk-logging-elasticsearch.adoc
+
+[id='efk-logging-elasticsearch-add-remove_{context}']
+= Changing the number of Elasticsearch nodes
+
+You can scale the number of nodes in Elasticsearch.
+
+For example, if you want to increase redundancy, and use the `FulRedundancy` or `MultipleRedundancy` policy, you can add additional data nodes to increase the number of shard replicas in your cluster.  
+
+A {product-title} Elasticsearch cluster can contain three node types, or roles:
```

Comment:
@ewolinetz Thoughts here?  I think we should exclude it all together as there is no way to modify via CL and I'm not certain how well the EO at this time supports modifying should we advise to go unmanged

Reply:
@mburke5678 +1 on this, we don't want to expose this level of detail to customers. currently it is possible for them to control that, but we intend to take that away and have the operator make these scaling/topology decisions for customers.